core/arch/proc.h

```diff
@@ -46,6 +46,7 @@
 
 /* exported for efficient access */
 extern size_t cache_line_size;
+extern size_t dcache_zva_size;
```

Comment:
Maybe note in a comment why we want to keep two separate variables for these, even though we expect them to be the same.